src/apiCalls.js

```diff
@@ -0,0 +1,12 @@
+export const fetchData = async () => {
+
+  let response = await fetch('https://rickandmortyapi.com/api/character')
+
+  if (!response.ok) {
```

Comment:
If you want to you can have response.status === 400 and throw specific errors for 400 or 500.

Reply:
ohhhhhh! Yes thank you! I will use this